src/test/rpc/JSONRPC_test.cpp

```diff
@@ -2586,6 +2588,7 @@ class JSONRPC_test : public beast::unit_test::suite
                         assert(get<1>(testFunc) == nullptr);
                         result = signFn(
                             req,
+                            1,
```

Comment:
I want to understand why we're not looping through all the API versions here. Is it because this PR only modified the response formats? 
I don't think the request format has been modified by this PR.